.github/workflows/operator-ci.yml

```diff
@@ -138,11 +138,11 @@ jobs:
           ./mvnw install -Poperator -pl :keycloak-operator -am \
               -Dquarkus.container-image.build=true \
               -Dquarkus.kubernetes.image-pull-policy=IfNotPresent \
-              -Doperator.keycloak.image=keycloak:${{ env.version_remote }} \
-              -Dquarkus.kubernetes.env.vars.operator-keycloak-image-pull-policy=Never \
-              -Dtest.operator.custom.image=custom-keycloak:${{ env.version_remote }} \
+              -Dkc-operator.keycloak.image=keycloak:${{ env.version_remote }} \
+              -Dquarkus.kubernetes.env.vars.kc-operator-keycloak-image-pull-policy=Never \
+              -Dtest.kc-operator.custom.image=custom-keycloak:${{ env.version_remote }} \
               --no-transfer-progress -Dtest.operator.deployment=remote \
-              -Dtest.operator.kubernetes.ip=$(minikube ip)
+              -Dtest.kc-operator.kubernetes.ip=$(minikube ip)
```

Comment:
```suggestion
              -Dtest.kc.operator.kubernetes.ip=$(minikube ip)
```

Reply:
On the test properties - test.operator.kubernetes.ip is no longer needed, and I meant to leave test.operator.custom.image alone as it (and test.operator.deployment) are specifically test properties.